addon-resizer/README.md

```diff
@@ -102,3 +102,146 @@ spec:
           configMap:
             name: nannyconfig
 ```
+
+## Addon resizer configuration
+
+To follow instructions in this section, set environment variable `ADDON_NAME` to
+name of your addon, for example:
+
+```
+ADDON_NAME=$ADDON_NAME
```

Comment:
This line does not seem to make sense, you might wanted to write:
ADDON_NAME=heapster

Please add a line at the beginning of this README listing all the places that you know where this addon-resizer is in use, e.g.
In GKE the addon-resizer is used to scale heapster and ...